barcode_dc/code/barcode_scan.py

```diff
@@ -101,12 +101,13 @@ def find_scanner(self):
 
                 try:
                     if dev.properties['ID_INPUT_KEYBOARD'] == "1":
-                        _, connection_point = dev.properties['ID_PATH'].split('-usb-')
+                        platform, connection_point = dev.properties['ID_PATH'].split('-usb-')
                         serial_option_1 = dev.properties['ID_SERIAL']
                         serial_option_2 = f"{dev.properties['ID_VENDOR_ID']}_{dev.properties['ID_MODEL_ID']}"
                         logger.info(
                             f"available: {serial_option_1} or "
-                            f"{serial_option_2} on connection point {connection_point.split(':')}")
+                            f"{serial_option_2} on connection point {connection_point.split(':')} "

```

Comment:
The default config (`connection_point=["0","1.3.1"]`) doesn't include a `:`. How is it meant to split by it? It is unclear what this is meant to do.

Reply:
that's not the connection point from the config file that is being split - it is the string from the os. Splitting the os string puts it into the form that is used in the config file. (OS string would be "0:1.3.1:<a bunch of other things>" which is split to ["0","1.3.1","<a bunch of other things>"] and then compared to find a match.